### PR TITLE
[WIP] AC-424:Do not require login location if no locations are configured

### PR DESCRIPTION
**Pull request by DefCon-007**

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
 import android.graphics.Color;
 import android.text.Editable;
 import android.text.TextWatcher;
+import android.util.Log;
```
[the remark this answers is not in the document]
Comment (dkayiwa) [reply openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.java]: @DefCon-007 did you see the above?

[the remark this answers is not in the document]
Comment (DefCon-007) [reply openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.java]: Hey @dkayiwa , I have fixed these issues already was thinking to push after testing with the kenya server.

[the remark this answers is not in the document]
Comment (dkayiwa) [reply openmrs-client/src/main/java/org/openmrs/mobile/listeners/watcher/LoginValidatorWatcher.java]: @DefCon-007 how did it go with the kenya server? 😊

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/login/LoginFragment.java [hunk 2]
```diff
@@ -353,7 +354,8 @@ private Snackbar createSnackbar(String message) {
     @Override
     public void setLocationErrorOccurred(boolean errorOccurred) {
         this.loginValidatorWatcher.setLocationErrorOccurred(errorOccurred);
-        mLoginButton.setEnabled(!errorOccurred);
+//        mLoginButton.setEnabled(!errorOccurred);
```
Comment (dkayiwa) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/login/LoginFragment.java]: Did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode

**Comment by dkayiwa** [comment]
@DefCon-007 do you mind following the guidelines at? https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by dkayiwa** [comment]
@DefCon-007 are you still working on this?

**Pull request closed without merging**